### PR TITLE
Improve pre-commit compliance

### PR DESCRIPTION
**Pull request by trexfeathers**
The introduction of pre-commit (#400) has highlighted several non-compliances. Seems simplest to fix them in a dedicated PR.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -68,21 +68,22 @@ def file_walk_relative(top, remove=''):
...
+    description      = "GRIB loading for Iris",                  # noqa: E221,E251,E501
+    long_description = long_description(),                       # noqa: E221,E251,E501
+    long_description_content_type = "text/x-rst",                # noqa: E221,E251,E501
+    url              = 'https://github.com/SciTools/iris-grib',  # noqa: E221,E251,E501
+    author           = 'UK Met Office',                          # noqa: E221,E251,E501
+    author_email     = 'scitools-iris@googlegroups.com',         # noqa: E221,E251,E501
+    license          = 'BSD',                                    # noqa: E221,E251,E501
+    platforms        = "Linux, Mac OS X, Windows",               # noqa: E221,E251,E501
+    keywords         = ['iris', 'GRIB'],                         # noqa: E221,E251,E501
+    classifiers=[                                                # noqa: E221,E251,E501
```
Comment (trexfeathers) [inline setup.py:86]: This section will be gone soon anyway (#387) so this seemed the quickest way to get it compliant.

File: docs/conf.py [hunk 2]
```diff
@@ -57,8 +58,7 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
-import iris_grib
```
Comment (trexfeathers) [inline docs/conf.py:60]: The actual Flake8 complaint was about where the `import` was positioned, but I figured I should just replicate what Iris does.

**Review by pp-mo — APPROVED** [review]
Amazed there is so little TBH!
Great to have this in.

**Comment by trexfeathers** [comment]
Thanks for the speedy action @pp-mo

**Pull request merged**